src/App.css

```diff
@@ -0,0 +1,20 @@
+#the-circle-of-sounds {
+  height: 50vw;
+  width: 50vw;
+  border-radius: 25vw;
+  margin: auto;
+  margin-top: 50px;
+}
+
+#background-sounds-container {
+  display: flex;
+}
+
+@keyframes App-logo-spin {
```

Comment:
I don't have the chance to see your deployed app yet, but I know this is the default CSS for the spinning React logo. I suspect it's not necessary here.